### PR TITLE
[1.1.x] Parity with 2.0.x in UBL and EEPROM

### PR DESCRIPTION
**Pull request by thinkyhead**
Catching up with the latest code mods. Please review.

### CODE REVIEW

**Comment by Roxy-3D** [comment]
```
-    static void manually_probe_remaining_mesh(const float&, const float&, const float&, const float&, const bool) _O0;
-    static void fine_tune_mesh(const float &rx, const float &ry, const bool do_ubl_mesh_map) _O0;
-    static bool g29_parameter_parsing() _O0;
```

The _O0's need to stay.   Otherwise GCC optimizes by in-lining the subordinate functions that G29 calls.   And instead of being smart and re-using stack space it keeps allocating more stack space for each inlined function.    AND...   the firmware goes crazy (WITH NO WARNING!!!) when the stack frame is exceeded.

It took 3 or 4 of us to find this problem.    But I did learn how to use GCC's object file dumper because of it.   

PS.   The code size hardly changes at all as a result of turning off optimization.   And speed of G29 just doesn't matter.    So, in this one case, there is no penalty for turning the optimization off.

**Comment by thinkyhead** [comment]
How is it that 2.0.x and 1.1.x differ on those lines? They should both need it or not need it.

**Comment by thinkyhead** [comment]
> And instead of being smart and re-using stack space it keeps allocating more stack space for each inlined function. AND... the firmware goes crazy (WITH NO WARNING!!!) when the stack frame is exceeded.

There might be some code-based ways to avoid that. We can look into it later.

File: Marlin/ubl.h [hunk 1]
```diff
@@ -112,16 +112,17 @@ class unified_bed_leveling {
...
     static mesh_index_pair find_furthest_invalid_mesh_point() _O0;
     static void reset();
     static void invalidate();
     static void set_all_mesh_points_to_value(const float);
     static bool sanity_check();
 
-    static void G29() _O0;                      // O0 for no optimization
-    static void smart_fill_wlsf(const float &); // O2 gives smaller code than Os on A2560
+    static void G29() _O0;                          // O0 for no optimization
+    static void smart_fill_wlsf(const float &) _O2; // O2 gives smaller code than Os on A2560
```
Comment (thinkyhead) [inline Marlin/ubl.h:124]: So this `_O2` goes away?

**Comment by thinkyhead** [comment]
Ok, they are now back in place.

**Pull request merged**